ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/BGP_PEER_GROUPS.yml

```diff
@@ -29,6 +29,7 @@ node_type_keys:
     default_evpn_encapsulation: mpls
 
 bgp_peer_groups:
+  use_separate_mlag_peer_group_for_overlay: true
```

Comment:
please add a test testing the `mlag_ipv4_overlay_peer` as well

Reply:
Added!